views/parts/events/row.php

```diff
@@ -43,10 +43,12 @@
 					<a href="#" data-nonce="<?php echo esc_attr( $event->nonce( 'run' ) ); ?>" data-event="<?php echo esc_attr( $event->hash ); ?>" class="run-event"><?php esc_html_e( 'Execute now', 'advanced-cron-manager' ); ?></a> |
 				</span>
 				<span class="pause">
```

Comment:
Even if the event cannot be paused, you still display the `<span class="pause">`, do we need it?

Reply:
You are right, span was removed